orchagent/copporch.cpp

```diff
@@ -489,6 +489,7 @@ task_process_status CoppOrch::processCoppRule(Consumer& consumer)
                 if (sai_status != SAI_STATUS_SUCCESS)
                 {
                     SWSS_LOG_ERROR("Failed to apply attribute:%d to trap group:%" PRIx64 ", name:%s, error:%d\n", trap_gr_attr.id, m_trap_group_map[trap_group_name], trap_group_name.c_str(), sai_status);
+                    handleSaiSetStatus(SAI_API_HOSTIF, sai_status);
```

Comment:
>handleSaiSetStatus [](start = 20, length = 18)

Check the return value?

Reply:
There are a few SAI calls that do not take any action in the case of SAI failures. For these ones, since our goal is to make it on-par with async mode at this stage, I am not checking the return value for now so that it has the same behavior as before.

Reply:
Added a check for the return value of the handling function and return only when it is not `task_success`